components/suggestion/suggestion.jsx

```diff
@@ -34,4 +35,10 @@ export default class Suggestion extends React.Component {
 
         this.props.onClick(this.props.term, this.props.matchedPretext);
     }
+
+    handleMouseMove = (e) => {
+        e.preventDefault();
```

Comment:
Do we need `preventDefault` here? I think we have it in the on click handler to stop an `#` from being added to the URL, but I don't think it does anything here

Reply:
Is there anything I can do? Whenever I try to merge `suggestion_store.jsx` is deleted and gives many errors.

Reply:
@s4kh, I've merged `master` into your branch -- if you pull down from your remote, you should see the conflict resolved. Let us know if this unblocks you :)